parser/tests/TPCH.test

```diff
@@ -1302,9 +1657,9 @@ WHERE
 ORDER BY
   s_name
 --
-ERROR: syntax error (8 : 13)
-  s_suppkey in (
-            ^
+ERROR: syntax error (23 : 9)
+        SELECT
```

Comment:
Question: does this parser error result from missing the sub-query support? Thanks!